docs/getting-started.md

```diff
@@ -9,23 +9,87 @@ This is a first draft of the tutorial, and may have errors or omissions.
+*  have a few pieces of software available; and
+*  understand a few concepts which might be new to you.
+
+## Running OpenSAFELY
+
+### Options for running OpenSAFELY
+
+In this guide, we've documented two different ways to work with OpenSAFELY:
+
+1. *Use an online environment where the needed software is already
+   installed for you*.
+
+     Services such as [Gitpod](https://gitpod.io) provide online
+     computing environments; Gitpod currently has a free plan with a
+     generous monthly usage limit for working with public code. We've
+     adapted our demonstration research study and this guide such that
+     you can follow along with Gitpod, should you choose to do so.
+
+     For this option, the only requirement is a modern web browser (e.g.
+     Chrome, Edge, Firefox, Safari).
+
+     You might prefer an online environment if one or more of the
+     following apply:
+
+     * you are investigating what OpenSAFELY offers and want to start
+       using it immediately;
+     * you do not have administrative access to install software on your
+       work computer;
+     * you are not sure about whether your computer supports Docker,
+       which OpenSAFELY uses;
+     * you want to try OpenSAFELY via a device other than a desktop or
+       laptop computer, e.g. a tablet.
+
+2. *Install the required software to your own computer*.
+
+     You might prefer a local installation if one or more of the following
+     apply:
+
+     * you already have the software required (Docker, Python and Git or
+       GitHub Desktop) installed;
+     * or you don't already have the required software installed, but
+       are comfortable installing and configuring these tools yourself;
+     * you want to have more control on the tools you use to develop
+       studies for OpenSAFELY.
+
+    The current local installation guide is aimed at Windows
+    users. Mac users should be able to follow along as well, with a few
+    hopefully-obvious alterations; see also the [macOS Install
+    Guide](install-macos.md)! We aim to integrate macOS instructions
+    into this guide in future.
+
+### Following this Getting Started guide
 
-Some of the software you'll install is so you can execute code on your computer
+!!!note
+    *Tabbed* sections like the example immediately below split the
+    configuration instructions into web browser and local Windows
+    installation.  Where these tabbed sections appear, select the
+    relevant tab and follow just the instructions for that section.
+
+=== "Web browser (online)"
+
+    These "Web browser" sections will explain how to use OpenSAFELY in a
+    web browser via the Gitpod service.
+
+=== "Windows (local)"
+
+    These "Windows" sections will explain how to use OpenSAFELY using
+    software installed directly on a Windows machine. These instructions
+    are not necessary to follow if you are using a web browser to run
+    OpenSAFELY, even if that web browser is running on Windows.
+
```

Comment:
I'll try changing it.

I find the end of tabbed sections difficult to work out at times. This is a bit of a problem with the theme we're using. Typically I've added headings after tabbed sections to try to demarcate it more.

The other option is that we override the theme CSS and do something like stick another line underneath.

Reply:
Yes. I'm in favour of a light CSS override, it probably makes sense. Though usually I see this in the concept of *code* snippets like this - it may look a bit funny for narrative snippets

![image](https://user-images.githubusercontent.com/211271/121313755-9c08c180-c8fe-11eb-9422-c14d2cc46ff8.png)

Reply:
Could you try something ugly like a coloured background using browser DOM manipulation and paste the results here for more discussion? That'd be good enough to have a sense of if it'd work, and then we could ask Tom to do something nice if we like it?

Reply:
![tabbed-css-override-background-color](https://user-images.githubusercontent.com/3818079/121359018-fae43000-c92a-11eb-96bd-113b9e357026.png)

Reply:
Going to mark as resolved for now, because we may move away from tabs completely. Otherwise, we need to revisit it.